### PR TITLE
Add English WNP for Firefox 86 [fix #9933]

### LINKED ISSUE #9933: WNP 86 for EN
## Description

/en-*/ page promoting Firefox for mobile, featuring the send-to-device widget.

DE and FR are being done separately in https://github.com/mozilla/bedrock/issues/9926 and other locales will get the evergreen page (which also promotes mobile).

---

#### Success Criteria
- [ ] Content is live before 23 February

### PR DESCRIPTION
**Pull request by craigcook**
## Description
Adds a What's New Page for Firefox 86.0 for English (en-US, en-CA, en-GB).

## Issue / Bugzilla link
#9933 

## Testing
http://localhost:8000/en-US/firefox/86.0/whatsnew/all/

English locales should get the new page, others should get the evergreen page. DE and FR are getting their own custom pages in https://github.com/mozilla/bedrock/pull/9932

### CODE REVIEW

**Comment by craigcook** [comment]
Do Not Merge for the moment, awaiting final signoff from stakeholders.

**Comment by craigcook** [comment]
Approved!

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx86-en.html
Comment (stephaniehobson) [inline bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx86-en.html:52]: Nit: these 2 `<h3>` should be `<h2>`s

**Review by stephaniehobson — APPROVED** [review]
R+ 📱

**Comment by craigcook** [comment]
Updated and added functional tests that I shamelessly stole from https://github.com/mozilla/bedrock/pull/9932

**Comment by craigcook** [comment]
Rebased to incorporate the DE and FR pages that just merged, we should make sure I didn't break something in the rebase and all tests pass (they do for me locally).

**Comment by craigcook** [comment]
Successful test run: https://gitlab.com/mozmeao/www-config/-/pipelines/258902340

File: tests/functional/firefox/whatsnew/test_whatsnew_86.py
Comment (stephaniehobson) [inline tests/functional/firefox/whatsnew/test_whatsnew_86.py:12]: 🔥

**Review by stephaniehobson — APPROVED** [review]
R+ ⏫

**Pull request merged**
